### PR TITLE
Add documentation to launch development server behind a proxy #603

### LINKED ISSUE #603: Need to add "index.html" to make dev server work
Hi,

In my dev environement I need to add "/index.html" to

- root (ex: localhost:3000/index.html)
- blog (ex: localhost:3000/blog/index.html)

Otherwise I get a server timeout error (the page does not defaults to "index.html").

Any idea why that might be happening?
Cheers.

### PR DESCRIPTION
**Pull request by sebelga**
Documentation on how to launch the development server behind a corporate proxy.

Fixes #603

### CODE REVIEW

**Comment by sebelga** [comment]
I am sorry but this didn't seem to have been included in the build. I might have missed something on the versioning. Let me know how to correctly update the version + my changes :smile:

**Comment by endiliey** [comment]
When you change the doc file it will be available on `next` documentation.

In local dev
http://localhost/docs/en/next/installation.html

In netlify
https://deploy-preview-688--docusaurus-preview.netlify.com/docs/en/next/installation.html

I don't have write access. Let's wait for @yangshun and @joelmarcey to see this PR and see what they say ☺

**Comment by JoelMarcey** [comment]
@endiliey is correct 👍 

New documentation will be in the `next` section of the documentation. So you click on the version number and then the link for the next version and you should see it.

We should probably add something about this to the `CONTRIBUTING.md` file

File: docs/getting-started-installation.md [hunk 1]
```diff
@@ -51,3 +51,12 @@ Running the Docusaurus initialization script, `docusaurus-init`, produces a runn
 1. Load the example site at http://localhost:3000. You should see the example site loaded in your web browser.
 
 ![](/img/getting-started-preparation-verify.png)
+
+## Launching the server behind a Proxy
```
Comment (JoelMarcey) [inline docs/getting-started-installation.md]: I think this should be a `###` or a `>` note.

File: docs/getting-started-installation.md [hunk 2]
```diff
@@ -51,3 +51,12 @@ Running the Docusaurus initialization script, `docusaurus-init`, produces a runn
...
+
+If you are behind a corporate proxy you need to disable it for the development server requests. It is easily done using the `NO_PROXY` environment variable.
```
Comment (JoelMarcey) [inline docs/getting-started-installation.md]: a comma (,) after "proxy"

**Comment by yangshun** [comment]
Looks great! Since the edits were minor, I fixed them and committed them. Thanks again!

**Comment by sebelga** [comment]
Great! thanks :+1:

**Pull request merged**